pkg/apis/servicecatalog/types.go

```diff
@@ -315,6 +322,10 @@ type ServicePlan struct {
 	// ServiceInstanceCredentialCreateParameterSchema is the schema for the parameters that
 	// may be supplied binding to an ServiceInstance on this plan.
 	ServiceInstanceCredentialCreateParameterSchema *runtime.RawExtension
+
+	// ServiceClassRef is a reference to the service class that
+	// owns this plan.
+	ServiceClassRef v1.LocalObjectReference
```

Comment:
This could be just an Owner Reference with Controller=true?

Reply:
It's merged, so we can do followups now.

What does `controller=true` mean?

Reply:
@MHBauer Kubernetes object could have multiple OwnerReferences, but only one of them could be a controller ("true owner").
IMO a dedicated `ServiceClassRef` makes sense there, and provides a better more transparent UX.
That said, I agree with @ash2k that we should also set OwnerReference to make sure there is no clash between different classes having plans with same UUIDs/names.
See pull request https://github.com/kubernetes-incubator/service-catalog/pull/979 for the example of how I set OwnerReference for secret owned by binding.

Reply:
See item 11 https://github.com/kubernetes/community/blob/master/contributors/devel/controllers.md
and https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/controller-ref.md for details.